src/components/notifications/content-notification/content-notification.js

```diff
@@ -1,9 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
+import { css } from '@emotion/core';
 import { ErrorIcon, WarningIcon, InfoIcon, CheckBoldIcon } from '../../icons';
 import filterDataAttributes from '../../../utils/filter-data-attributes';
-import styles from './content-notification.mod.css';
+import vars from '../../../../materials/custom-properties';
+
+const getIconContainerBackgroundColour = props => {
+  switch (props.type) {
+    case 'error':
+      return vars['--color-red'];
+    case 'info':
+      return vars['--color-blue'];
+    case 'warning':
+      return vars['--color-orange'];
+    case 'success':
+      return vars['--color-green'];
```

Comment:
Wait. This is not to way to go now?

Reply:
Why not?

Reply:
I missed that we made a breaking change 🤦‍♂️